### PR TITLE
made installer paramiterized and added timeout as its causing issues

### PR DESCRIPTION
**Pull request by jmunnik**
ProcessId: 2292
app_name: C:\Windows\system32\cmd.exe
command_line: cmd /c "start "" /wait "c:\chef\cache\mcafee virusscan enterprise\setupvse.exe"   ADDLOCAL=ALL RUNAUTOUPDA
TESILENTLY=True REMOVE=LotusNotesScan REBOOT=R /qn & exit %%%%ERRORLEVEL%%%%"
timeout: 600

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -24,3 +24,5 @@
 default['mcafee']['virusscan']['package_name']  = 'McAfee VirusScan Enterprise'
 default['mcafee']['virusscan']['url']           = ''
 default['mcafee']['virusscan']['checksum']      = ''
+default['mcafee']['virusscan']['options']       = 'ADDLOCAL=ALL RUNAUTOUPDATESILENTLY=True REMOVE=ScriptScan,LotusNotesScan,EmailScan REBOOT=R /qn'
+default['mcafee']['virusscan']['installer_timeout']   = '900'
```
Comment (svanharmelen) [inline attributes/default.rb]: Please align all values one space after the longest key :wink:

**Comment by svanharmelen** [comment]
@jmunnik small styling thingy, otherwise LGTM.

File: attributes/default.rb (hunk 1 above)
Comment (jmunnik) [inline attributes/default.rb]: @svanharmelen ; done can you have a look and merge

**Comment by svanharmelen** [comment]
👍

**Pull request merged**